src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString.trim()
+    .split(';')
+    .map(prop => prop.split(':').map(item => item.trim()))
+    .filter(item => item[0] !== '')
+    .forEach((item) => {
+      result[item[0]] = item[1];
+    });
```

Comment:
Try to use destructuring assignment here.
This will improve the readability of the code.

```suggestion
    .forEach(([key, value]) => {
      result[key] = item[value];
    });
```

Reply:
👌made it, looks much better))) thanks!